penguin/core.py

```
@@ -4318,6 +4322,101 @@ def finalize_streaming_message(
+                    resolved_session_id or resolved_conversation_id
+                )
+                event_data["conversation_id"] = resolved_conversation_id
+            elif resolved_session_id:
+                event_data["session_id"] = resolved_session_id
+                event_data["conversation_id"] = resolved_session_id
+            else:
+                event_data["session_id"] = "unknown"
+                event_data["conversation_id"] = "unknown"
+            event_data["agent_id"] = resolved_agent_id
+            event_data = self._filter_internal_markers_from_event(event_data)
+            asyncio.create_task(self.emit_ui_event(event.event_type, event_data))
```

Ruff (0.15.12)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)